public-ui/src/app/data-browser/views/survey-view/components/survey-answer-react.component.tsx

```diff
@@ -97,12 +98,14 @@ const SurveyAnswerRowComponent = (class extends React.Component<SurveyRowProps,
     }
 
 
-    openDrawer() {
+    openDrawer(answerValueString) {
         this.setState({
-            drawerOpen: !this.state.drawerOpen
+            drawerOpen: (answerValueString !== 'Did not answer') ? !this.state.drawerOpen : false
         });
         if (this.props.hasSubQuestions === '1' && !this.state.subQuestions.length) {
             this.getSubQuestions();
+        } else {
+            this.processResults([this.props.question]);
```

Comment:
Added this in here to also process results for the answers that do not have sub-questions.